src/xmipp/libraries/data/fftwT.cpp

```diff
@@ -23,7 +23,8 @@
  *  e-mail address 'xmipp@cnb.csic.es'
  ***************************************************************************/
 
-#include "fftwT.h"
+#include "../data/fftwT.h"
```

Comment:
include "fftcW.h" does not work? must search in the same folder "data"

Reply:
This is because the Fourier library was moved to data. It is not a method, so I moved to data. It will be shared between SPA and tomo in a future

I will fix the bugs now

Reply:
ammm ok, thanks